docs/hooks/appointment-booking.md

```diff
@@ -0,0 +1,237 @@
+# `appointment-booking`
+
+| Metadata | Value
+| ---- | ----
+| specificationVersion | 1.0
+| hookVersion | 1.0
```

Comment:
Is a 1.0 version for this hook appropriate? Regardless, you should also specify a [hookMaturity](https://cds-hooks.org/hooks/template/).

```suggestion
| hookVersion | 1.0
| hookMaturity | 0 - Draft 
```